src/common/error-messages.ts

```diff
@@ -9,5 +9,5 @@ export enum SendFormErrorMessages {
   MustSelectAsset = 'You must select a valid token to transfer',
   TooMuchPrecision = '{token} can only have {decimals} decimals',
   MemoExceedsLimit = 'Memo must be less than 34-bytes',
-  AdjustedFeeExceedsBalance = 'AdjustedFeeExceedsBalance',
+  AdjustedFeeExceedsBalance = 'The fee added now exceeds your current STX balance.',
```

Comment:
This was used in two places so I felt like the message could just be moved here?